Test/UnitTest1.cs

```diff
@@ -1,8 +1,69 @@
+using Training;
 namespace Test {
    [TestClass]
    public class UnitTest1 {
       [TestMethod]
-      public void TestMethod1 () {
+      public void PassTest () {
+         Assert.AreEqual (double.Parse ("0"), MyDouble.Parse ("0"));

```

Comment:
- Repeated statements but just with different inputs. Could have used loop to assert outputs for a list of input values.

- Possible mistakes while providing LHS and RHS input values. Could have eliminated by using variables.
```suggestion
         Assert.AreEqual (double.Parse (inputVal), MyDouble.Parse (inputVal));
```